packages/client/components/StageTimerDisplayGauge.tsx

```diff
@@ -30,11 +31,33 @@ const Gauge = styled('div')<{isTimeUp: boolean; isDesktop: boolean}>(({isTimeUp,
   userSelect: 'none'
 }))
 
+const useSoundEffect = (isTimeUp: boolean) => {
+  // avoids playing when navigating to
+  // a page when the timer is already up.
+  const hasPlayed = useRef(false)
+
+  useEffect(() => {
+    hasPlayed.current = isTimeUp
+  }, [])
+
+  useEffect(() => {
+    if (isTimeUp && !hasPlayed.current) {
+      new Audio(sound).play()
```

Comment:
+1 this is creating the audio element everytime the hook is called so we could move it out of the useEffect and memoize it, say on line 35: `const audioElement = useMemo(() => new Audio(sound), [])`

It's a small file though so not a big deal at all. It might matter if we reuse this hook with larger audio files

Reply:
Agreed, I will update that.